Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -281,7 +283,13 @@ void Renderer::DrawDebugText()
   if (SConfig::GetInstance().m_ShowInputDisplay)
   {
     final_cyan += Movie::GetInputDisplay();
```

Comment:
What do you think about adding the amount of lines as a second return value of GetInputDisplay?

Reply:
Do you mean something like this?